internals/webpack/webpack.ssr.babel.js

```diff
@@ -0,0 +1,88 @@
+// Important modules this config uses
+const path = require('path');
+const webpack = require('webpack');
+const nodeExternals = require('webpack-node-externals');
+
+const outputPath = path.join(process.cwd(), 'server', 'middlewares');
+
+module.exports = {
+  name: 'server',
+  target: 'node',
+  externals: [nodeExternals()],
+  entry: [
+    path.join(process.cwd(), 'app/serverSideRenderAppToStringAtLocation.js'),
+  ],
+  output: {
+    path: outputPath,
+    filename: 'generated.serverSideRenderAppToStringAtLocation.js',
+    publicPath: '/',
+    libraryTarget: 'commonjs2',
+  },
+  module: {
+    loaders: [{
+      test: /\.jsx?$/, // Transform all .js files required somewhere with Babel
+      loader: 'babel',
+      exclude: /node_modules/,
+      query: {
+        only: ['app'],
+        presets: [
+          ['es2015', { modules: false }],
+          'react',
+          'stage-0',
+        ],
+        plugins: [
+          'transform-react-remove-prop-types',
+          'transform-react-constant-elements',
+          'transform-react-inline-elements',
+        ],
```

Comment:
Why do we have a separate config here, can't this just use the `babelConfig` from the `package.json`?